php/strict/ByteBuffer.php

```diff
@@ -0,0 +1,371 @@
+<?php declare (strict_types=1);
```

Comment:
Is this file duplicated from the non-strict version? Is there no better way to do this?